index.css

```diff
@@ -0,0 +1,107 @@
+:root {
+    --primary-color: #002D62;
+    --secondary-color: white;
+    --primary-shade-color:  #194271; 
+    --font-family: sans-serif;
+    font-size: 16px; 
+}
+
+body {
+    background-color: var(--primary-color) ;
+}
+
+.categories {
+    display: grid;
+    grid-template-columns: repeat(3,auto) ; /* create a 3 column layout of equal width **/
+    grid-auto-rows: auto;
+    grid-auto-columns: auto;
+    grid-gap: 5px;
+    
+    justify-content: center;
+    align-content: center;
+    grid-auto-flow: column;
+    padding: 0;
+    margin: 0;
+   margin-top: 10%;
+}
+
+.question-container {
+    width: 100px;
+    color: var(--secondary-color);
+}
+
+.question-container button {
+    width: 100%;
+    color: var(--secondary-color);
+    border-radius: 25px;
+}
+
+.text-align-center {
+  text-align: center;
+} 
+
+.center {
+  margin: auto;
+}
+
+.heading  {
+  font-size: 30px;
+  font-weight: bold;
+  margin-top: 1rem;
+}
+
+.sub-heading {
+  font-size: 25px;
+  font-weight: small;
+  margin-top: 1rem;
+}
+
+.card {
+    background-color: var(--primary-shade-color);
+    border-radius: 12px;
+    min-width: 400px;
+    font-size: 1rem;
+  }
+  
+.quizHeading {
+    text-align: center;
+    color: var(--secondary-color);
+    font-size: 2rem;
+}
+
+.card a {
+  text-decoration: none;
+}
+
+.card img {
+  width: 100%;
+  height: 40%;
+}
+
+.card:hover {
+  box-shadow: 0 8px 16px 0 rgba(0,0,0,0.2);
+}
+  
+.card-content {
+  padding: 2px 16px;
+  color: var(--secondary-color);
+}
+
+.options {
+  display: grid;
+  grid-template-columns: 1fr 0.5fr;

```

Comment:
You can use 2fr 1fr here for better readability. No difference of course, just easier to make out that first row is twice the width of the second.